app/workers/education_form/forms/va0994.rb

```diff
@@ -51,7 +51,7 @@ def applicant_ssn
     end
 
     def bank_routing_number
-      if @applicant.bankAccount.present? && @applicant.bankAccount.routingNumber.present?
+      if @applicant.bankAccount&.routingNumber.present?
```

Comment:
Sorry, if I wasn't initially clear (the github `suggest` feature only let's you suggest a change on a single line and not multiple lines).  I was suggesting changing the whole `if` statement:
```
if @applicant.bankAccount&.routingNumber.present?
        @applicant.bankAccount.routingNumber
      else
        value_or_na(@applicant.prefillBankAccount&.bankRoutingNumber)
      end
```
This way you don't need the
```
else
        'N/A'
```
because it is taken care of by `value_or_na`, which will return 'N/A'

Same logic applies for the other refactors.